### DISCUSSION TITLE
Up Stream Project for Xlink and POPS

### DISCUSSION THREAD

**Discussion by s3anami** [retronas/discussions#76] [Ideas]
Weren't sure if you were aware of this upstream project that could help with your implementation of those ideas
https://github.com/toolboc/psx-pi-smbshare

**Comment by sairuk** [comment]
Thanks, yep we ran across this one the other day, I'd really like to get xlink in. Always great to see community interest.

**Reply by FlotterCodername** [reply]
I'm trying to set up a combo of that psx-pi installer script and retronas right now.
What I basically want is that my PS2 and retronas-Pi3 are connected via RJ45 and both are integrated into the home network via the Pi3 wifi connection. PS2 gets very reliable bandwidth via Ethernet while the MiSTer/Recalbox etc. use retronas via Wi-Fi, which is fast enough for them in my case.

Would be really great to have this in retronas! Yesterday, I tried to set a bridge like that up manually and accidentally configured the Pi out of my Wi-Fi network 😆

Relates to retronas/retronas#76
